Initial cleanup

rylandg opened the pull request:
I removed the `src` directory (it's a trap for users) and hid the start file. I also added a basic README that explains the importance and rationale behind the project.

vogre commented on .start-local-express.js:
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
```
I think this could be in core `reshuffle`: opened https://github.com/reshufflehq/reshuffle/pull/317.

bergundy replied:
Nice @vogre

vogre commented on package.json:
```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
-    "start": "node src/start.js",
+    "start": "node .start-local-express.js",
```
can be `"start": "reshuffle-local-server"`.

rylandg commented:
@vogre @bergundy I've switched to the new versions, replaced the hidden file added a license and .gitignore. I will hold off on adding CircleCI and linting for a separate commit. Thanks for awesome quick addition Vlad.
